Makefile

```diff
@@ -48,6 +73,15 @@ ${BOARD}.img: boot/boot-${BOARD}.scr $(ROOTFS)
 	rm -f $@
 	tar -Szcf $@ $<
 
+## Convenience, avoiding the need to repeat yourself about which board
+img: $(BOARD).img
+tar: $(BOARD).tar
+tgz: $(BOARD).tar.gz
+
+dd: $(BOARD).img
+	sudo dd if=$(BOARD).img of=/dev/mmcblk0 bs=4096
```

Comment:
Hm - is this always /dev/mmcblk0? Seems a bit dangerous to just overwrite it here...